Equipo2Hosteleria/src/main/java/Vista/PanelPedidos.java

```diff
@@ -0,0 +1,252 @@
+package Vista;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.ButtonGroup;
+import javax.swing.JButton;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import Controlador.ControladorPanelPedidos;
+import Modelo.Producto;
+import Modelo.Modelo;
+
+import java.awt.Color;
+import java.awt.Font;
+import javax.swing.JComboBox;
+import javax.swing.JTextPane;
+import javax.swing.SwingConstants;
+import javax.swing.border.EmptyBorder;
+import javax.swing.event.ChangeEvent;
+import javax.swing.event.ChangeListener;
+import javax.swing.JRadioButton;
+
+@SuppressWarnings("serial")
+public class PanelPedidos extends JPanel implements ChangeListener {
+
+	private JButton btnVolver;
+	private JButton btnDesconectarse;
+	private JButton btnPasarACaja;
+	private JButton btnAadirAlCarro;
+	private JButton btnListaCompra;
+	private JComboBox comboBox_Productos;
+	private JComboBox comboBox_Cantidad;
+	private JLabel lblDireccin;
+	private JPanel contentPane;
+	private JRadioButton rdbtnRecogerEnEstablecimiento;
+	private JRadioButton rdbtnEntregaADomicilio;
+	private ButtonGroup grupoBotones;
+	private JTextPane textPane_Direccion;
+	private ControladorPanelPedidos controladorPanelPedidos;
+
+	
+	public PanelPedidos(ControladorPanelPedidos controladorPanelPedidos)
+	{
+		this.controladorPanelPedidos = controladorPanelPedidos;
+		
+		setLayout(null);
+		
+		JLabel lblPedidos = new JLabel("Pedidos");
+		lblPedidos.setHorizontalAlignment(SwingConstants.CENTER);
+		lblPedidos.setFont(new Font("Tahoma", Font.BOLD, 15));
+		lblPedidos.setBounds(38, 24, 362, 14);
+		add(lblPedidos);
+		
+		btnVolver = new JButton("Volver");
+		btnVolver.setBounds(38, 235, 89, 23);
+		add(btnVolver);
+		
+		comboBox_Productos = new JComboBox();
+		comboBox_Productos.setBounds(38, 84, 173, 22);
+		add(comboBox_Productos);
+		
+		Producto Producto1 = new Producto("Garbanzos","Comida","30-01-2021",5,5);	
+		Producto Producto2 = new Producto("Paella","Comida","31-01-2021",5,5);	
+		Producto Producto3 = new Producto("Bacalao","Comida","02-02-2021",5,5);	
+		Producto Producto4 = new Producto("Hamburguesa","Comida","30-01-2021",5,5);	
+		Producto Producto5 = new Producto("Cerveza","Bebida","01-02-2021",5,5);	
```

Comment:
Es mejor que no creéis los datos de pega en la vista, porque sino no vais a poder testearlo. Pasadlo al modelo.

Reply:
si, esto creo que lo comentamos en clase crearos en el modelo una clase para obtener los datos y meterlos en una estructura de datos que a través del controlador le paséis a la vista.

Reply:
Ahora los datos se crean en el modelo y se pasan a través del controlador.